distributed/scheduler.py

```diff
@@ -4009,8 +4009,10 @@ async def log_errors(func):
         )
 
         self.status = Status.closing
-
-        logger.info("Scheduler closing...")
+        if reason:
+            logger.info("Scheduler closing due to %s...", reason)
+        else:
+            logger.info("Scheduler closing...")
```

Comment:
nit: This will make it more obvious if the reason is not propagated.
```suggestion
        logger.info("Scheduler closing due to %s...", reason or "unknown reason")
```